{"name": "test_login_fail[测试用户名正确, 密码为空用户名:linux超密码期望值{'code': 1002, 'msg': '密码不能为空', 'data': None}]", "status": "passed", "description": "测试用户名或密码错误", "steps": [{"name": "输入用户名", "status": "passed", "parameters": [{"name": "user", "value": "'linux超'"}], "start": 1611730814381, "stop": 1611730814381}, {"name": "输入密码", "status": "passed", "parameters": [{"name": "pwd", "value": "''"}], "start": 1611730814381, "stop": 1611730814381}], "parameters": [{"name": "data", "value": "{'case': '用户名正确, 密码为空', 'user': 'linux超', 'pwd': '', 'expected': {'code': 1002, 'msg': '密码不能为空', 'data': None}}"}], "start": 1611730814381, "stop": 1611730814381, "uuid": "f966bfe6-f4ba-4f77-a34b-2270d203ecae", "historyId": "c580a07436c5ed792d6b51c0db9d5092", "testCaseId": "aa77bd5a391f291f3c0ff3906d95c794", "fullName": "pytest_allure.test_allure_feature.TestLogin#test_login_fail", "labels": [{"name": "story", "value": "测试登录失败"}, {"name": "severity", "value": "critical"}, {"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "pytest_allure"}, {"name": "suite", "value": "test_allure_feature"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "PC-20200914AAyx"}, {"name": "thread", "value": "9848-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "pytest_allure.test_allure_feature"}]}